docs/workforce-management-framework/topics/con-security-introduction.adoc

```diff
@@ -0,0 +1,49 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} server gives developers a choice one of the following authentication and authorisation providers
+link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} is build around security interface, which abstracts from any security libraries.
```

Comment:
"....is built around an interface which abstracts the developer away from any underlying authentication and authorisation libraries."

Reply:
"....is built around a security interface which abstracts the developer away from any underlying authentication and authorisation libraries."